src/components/app-components/Navbar.tsx

```diff
@@ -45,12 +45,14 @@ class Navbar extends React.Component {
 
                     <Box className={'navbar-items'} id={'navbar-right'}>
                         <a>
-                            Login
+                            Log In
```

Comment:
Log In - is a verb, Sign Up is also a verb, so I changed them. Signup is a noun, same with Login page